lib/ansible/modules/cloud/azure/azure_rm_subnet.py

```diff
@@ -64,8 +62,8 @@
         description:
             - The reference of the RouteTable resource.
             - It can accept both a str or a dict.
```

Comment:
I think this description and the description of the `security_group` option above should match:

```suggestion
```